Makefile

```diff
@@ -43,13 +43,20 @@ else
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone\""' vagrant up --provision
 endif
 
-up: update-box custom_ca
+up: check-params update-box custom_ca
 ifeq ($(GITHUB_ACTIONS),true) # Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
-	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone_git\""' vagrant up --provision
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone_git repo=$(repo) branch=${branch} user=${user} token=${token}\""' vagrant up --provision
 else
-	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone_git\""' vagrant up --provision
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone_git repo=$(repo) branch=${branch} user=${user} token=${token}\""' vagrant up --provision
 endif
 
+check-params:
+	@[ "${repo}" ] || ( echo ">> The parameter repo is not defined. repo=<GitHub-repository, use HTTPS>" )
+	@[ "${branch}" ] || ( echo ">> The parameter branch is not defined. branch=<branch to checkout and track>")
+	@[ "${user}" ] || (  echo ">> The parameter user is not defined. user=<GitHub username>")
+	@[ "${token}" ] || ( echo ">> The parameter token is not defined. token=<personal token from GitHub>" )
+	@[ "${repo}" ] && [ "${branch}" ] && [ "${user}" ] && [ "${token}" ]|| (echo "See README.md for more details and example: https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/master/example/standalone_git/README.md" ; exit 1 )
+
 test-standalone: clean up-standalone
 
 test: clean up
```

Comment:
I think you have to add params here as well!

Reply:
It has not been necessary when I tested the code, just write the following in CLI:
```
make test repo=https://github.com/hannemariavister/versioned_flows.git branch=master user=testuser token=thisisnotmypersonalgithubtoken
```

Reply:
Maybe should add how to run `make test` in README, what do you think?

Reply:
I agreed with @karimacherradi , could you write a few lines how to run make test in Readme

Reply:
Maybe Makefile is the place where this documentation should be?

Reply:
@AditiSharma77  @karimacherradi  I have updated documentation in [Makefile](https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/issue_35_arguments_makefile_vault/Makefile) and [README](https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/issue_35_arguments_makefile_vault/example/standalone_git/README.md) (according to what we talked about). Could you please read and give 👍 or 👎  ?